### DISCUSSION TITLE
Isit possible to have fans spin-up at a high temp but not spin-down until a lower temp??

### DISCUSSION THREAD

**Discussion by shrewdlogarithm** [Rem0o/FanControl.Releases#1603] [Q&A] [answered]
I like my fans to remain at a relatively low speed until my GPU hits - say - 70 and then they build speed from that point 

This I can easily do - however...

I'd like my fans to remain at a higher speed until they get the temp back into - say - the 50s - where it normally would be when not using the GPU...

Can I do this somehow??

Currently I either have to have the fans spinning-up at relatively low temps (so they bring the card back down to that temp) OR they slow down when the GPU goes below - say - 70 and then the card takes ages to drop back to 'normal' temps (when a minute-or-so of higher fanspeed would achieve that)

Is this doable somehow??

Thanks!

**Comment by Rem0o** [comment, marked as answer]
Different ways to do it. 

1) Mix, using max function, a graph and a trigger fan curve together, or just use a trigger. Idle temp of your trigger should be the temp at which it will start dropping the fan speed. 
2) In your graph, set hysteresis on the way down only, with a 20 degree hysteresis.

**Reply by shrewdlogarithm** [reply]
Thanks for that - I'm sure I can work out exactly what I need from here ;) :)
